src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentPrice = 40;
+
+  if (days < 3) {
+    return (days * dayRentPrice);
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * dayRentPrice) - 20;
+  }
+
+  if (days) {
+    return (days * dayRentPrice) - 50;
+  }
```

Comment:
```suggestion
  return (days * dayRentPrice) - 50;
```
if is redundant here, since `days` is an obligate  argument in the function (always defined)